configs/config.go

```diff
@@ -15,11 +16,27 @@ type AppConfig struct {
 
 var appConfig *AppConfig
 
-func GetAppConfig() *AppConfig {
-	err := godotenv.Load(".env")
+const projectDirName = "testGoApi"
```

Comment:
is there a way to get this from `go API` ?

Reply:
Well during the implementation I have tried to find a way to get it but unfortunately couldn't find a way